contrib/autocomplete/bash_autocomplete

```diff
@@ -6,7 +6,7 @@ _cli_bash_autocomplete() {
      local cur opts base
```

Comment:
Changed this to generate the static bash completion once to not require to load the cilium binary for every attempted completion.